### PR TITLE
Settings changes

### PR DESCRIPTION
**Pull request by KlaraMeyer0**

### CODE REVIEW

File: apps/client/src/app/App.tsx [hunk 1]
```diff
@@ -19,8 +19,6 @@ import ViewAll from '../../../../libs/client/ui/src/lib/view-all/view-all.js';
 const Stack = createNativeStackNavigator();
 
 export const App = () => {
-  const [whatsNextYCoord, setWhatsNextYCoord] = useState<number>(0);
-  const scrollViewRef = useRef<null | ScrollView>(null);
```
Comment (u20437863-James-Butler) [inline apps/client/src/app/App.tsx:23]: Why are these removed? I'm not sure what this PR is trying to do

Comment (KlaraMeyer0) [reply apps/client/src/app/App.tsx:23]: Those are leftovers from their original placeholder app that used to be there

Comment (KlaraMeyer0) [reply apps/client/src/app/App.tsx:23]: My messages are here

Comment (u20437863-James-Butler) [reply apps/client/src/app/App.tsx:23]: Ok, and the unit test stuff that was commented out?

Comment (KlaraMeyer0) [reply apps/client/src/app/App.tsx:23]: Yeah beacuse there is not a proper unit test yet. At the moment, i'm working on getting it onto the emulator

Comment (u20437863-James-Butler) [reply apps/client/src/app/App.tsx:23]: Ok

**Review by u20437863-James-Butler — APPROVED** [review]

**Comment by u20437863-James-Butler** [comment]
You can merge
